jobs.md

```diff
@@ -4,7 +4,11 @@ Come and work with us!
 
 ## Upstream team
 
-Interested in working on open source and containers at a leading hyper-scale cloud? You can help build the future of Kubernetes at Microsoft! Stay tuned for future
-openings.
+Interested in working on open source and containers at a leading hyper-scale cloud? You can help build the future of Kubernetes at Microsoft!
 
-[PLACEHOLDER]
\ No newline at end of file
+### Australia
+Hiring across levels 59-64
```

Comment:
Do we want to mention the titles here instead of levels? most people probably don't understand these numbers unless they have been at the company.
```suggestion
Hiring across all levels from early in career to senior software engineer.
```